src/wemmick/utils.py

```diff
@@ -14,3 +15,36 @@ def file_relative_path(dunderfile, relative_path: str):
     H/T https://github.com/dagster-io/dagster/blob/8a250e9619a49e8bff8e9aa7435df89c2d2ea039/python_modules/dagster/dagster/utils/__init__.py#L34
     """
     return os.path.join(os.path.dirname(dunderfile), relative_path)
+
+
+def get_scheme(path: str):
+    return urlparse(path).scheme
+
+
+def check_file_exists(path):
+    if not os.path.isfile(path):
+        raise ValueError(
+            f"File {path} was not found. Please check the path and try again."
+        )
+
+
+def get_file(path: str):
+    scheme = get_scheme(path)
+    if scheme == "s3":
+        # TODO: add AWS S3 support
```

Comment:
id rather not have todos like this in the code ...  say we had a ci action that created kanban cards for all todos (which would be quite nice) - we wouldnt want to create that card for maybes.

Reply:
I will remove these